### PR TITLE
Update changelog for 2.3.0 release

### PR DESCRIPTION
**Pull request by jeremymeng**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## 2.3.0 - UNRELEASED
+## 2.3.0 - 2021-03-26
```
Comment (ramya-rao-a) [inline Changelog.md:3]: Release is today or monday...?

Comment (jeremymeng) [reply Changelog.md:3]: I can do it today.

**Review by ramya-rao-a — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (jeremymeng) [inline Changelog.md:3]:
```suggestion
## 2.3.0 - 2021-03-29
```
well it's getting late. Monday then.

File: Changelog.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 2.3.0 - 2021-03-26
+## 2.3.0 - 2021-03-29
```
Comment (ramya-rao-a) [inline Changelog.md:3]: haha, thought so :)

**Pull request merged**